app/assets/javascripts/discourse/lib/transform-post.js.es6

```diff
@@ -121,6 +121,7 @@ export default function transformPost(currentUser, site, post, prevPost, nextPos
   postAtts.userCustomFields = post.user_custom_fields;
   postAtts.topicUrl = topic.get('url');
   postAtts.isSaving = post.isSaving;
+  postAtts.topicTags = topic.tags;
```

Comment:
This seems fine to me but we should only assign this if tags are enabled on the site.

Reply:
wait, why do we need this on every single post? we don't even have category from what I can see.

something does not feel right here if we are passing tags around but not category. @eviltrout thoughts?